UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -868,3 +902,13 @@ private void UpdateIcons()
 		}
 	}
 }
+
+
+[Flags]
+public enum TramuticDamageTypes
+{
+	NONE = 0,
+	SLASH = 1 << 0,
+	PIERCE = 2 << 1,
+	BURN = 4 << 2
```

Comment:
4 ? e.e

Reply:
idk it only decided to work after doing 4